rspec-core/rspec-core.gemspec

```diff
@@ -9,16 +9,16 @@ Gem::Specification.new do |s|
   s.license     = "MIT"
   s.authors     = ["Steven Baker", "David Chelimsky", "Chad Humphries", "Myron Marston"]
   s.email       = "rspec@googlegroups.com"
-  s.homepage    = "https://github.com/rspec/rspec-core"
+  s.homepage    = "https://github.com/rspec/rspec/tree/main/rspec-core"
```

Comment:
I think these should be `https://rspec.info`

Reply:
https://github.com/rspec/rspec/pull/157/commits/b6bde46d75adce43002a9941c3d73107cac7e33b